backpack.tf extended sorting.user.js

```diff
@@ -1268,4 +1268,246 @@ class</a></li>
+        splitPath[3] = splitPath[3].replace(ksTierDict[currentTier],"");
+        if(tier > 0){
+            splitPath[3] = ksTierDict[tier].concat("",splitPath[3]);
+        }
+        let tieredPath = splitPath.join("/");
+        return tieredPath;
+    }
+
+    class itemData{
+        constructor(headerItem){
+            this.name = headerItem.dataset.base_name;
+            this.defindex = headerItem.dataset.defindex;
+            this.killstreak_tier = headerItem.dataset.ks_tier ? headerItem.dataset.ks_tier : 0;
+            this.aussie = headerItem.dataset.australium ? headerItem.dataset.australium : 0;
+            this.slot = headerItem.dataset.slot ? headerItem.dataset.slot : "none";
+            this.idreplacedict = {
+                13 : 200,	// Scattergun
+                294 : 160,	// Lugermorph
+                833 : 812,	// Flying Guillotine
+                1071 : 264,	// Golden Frying Pan
+                21 : 208,	// Flame Thrower
+                30474 : 208,	// Nostromo Napalmer
+                834 : 813,	// Neon Annihilator
+                20 : 207,	// Stickybomb Launcher
+                1 : 191,	// Bottle
+                266 : 132,	// Headtaker
+                15 : 202,	// Minigun
+                832 : 811,	// Huo-Long Heater
+                5 : 195,	// Fists
+                169 : 197,	// Golden Wrench
+                17 : 204,	// Syringe Gun
+                29 : 211,	// Medi Gun
+                8 : 198,	// Bonesaw
+                14 : 201,	// Sniper Rifle
+                16 : 203,	// SMG
+                24 : 210,	// Revolver
+                4 : 194,	// Knife
+            }
+            this.nameiddict = {
+                "Scattergun" : 200,
+                "Force-A-Nature" : 45,
+                "Holy Mackerel" : 221,
+                "Bat" : 190,
+                "Rocket Launcher" : 205,
+                "Black Box" : 228,
+                "Shotgun" : 199,
+                "Flame Thrower" : 208,
+                "Backburner" : 40,
+                "Flare Gun" : 39,
+                "Axtinguisher" : 38,
+                "Grenade Launcher" : 206,
+                "Stickybomb Launcher" : 207,
+                "Chargin' Targe" : 131,
+                "Eyelander" : 132,
+                "Minigun" : 202,
+                "Gloves of Running Urgently" : 239,
+                "Frontier Justice" : 141,
+                "Crusader's Crossbow" : 305,
+                "Medi Gun" : 211,
+                "Bonesaw" : 198,
+                "Ubersaw" : 37,
+                "Sniper Rifle" : 201,
+                "Huntsman" : 56,
+                "SMG" : 203,
+                "Revolver" : 210,
+                "Ambassador" : 61,
+                "Knife" : 194,
+            }
+        }
+
+        getItemName(){
+            let itemName = this.name;
+            if(this.aussie == 1){
+                itemName = "Australium".concat(" ", itemName);
+            }
+            return itemName;
+        }
```

Comment:
(nitpick)
Why not just:

```suggestion
        getItemName(){
            return (this.aussie == 1) ? "Australium " + this.name : this.name;
        }
```

?